package-mixins.js

```diff
@@ -1,20 +1,36 @@
 var extend = require("extend");
+var { NativeAppEventEmitter } = require('react-native');
```

Comment:
In the CodePush repo we use double quotes here and 4 spaces per indent.

Let's do the same here.